devicon.json

```diff
@@ -14966,4 +14685,25 @@
         "color": "#443e38",
         "aliases": []
     }
+    {
+        "name": "gamemaker",
```

Comment:
the name in devicon.json needs to match the folder- and file-names. According to the brand guidelines, the "Studio" part is not used anymore, i suggest to rename folder and files to `gamemaker`

> We no longer use the names GameMaker:Studio or GameMaker Studio 2.